### PR TITLE
Fix printing of kernel args

### PR DESCRIPTION
**Pull request by naoyam**
And some cleanups

### CODE REVIEW

File: torch/csrc/jit/codegen/cuda/executor.cpp [hunk 1]
```diff
@@ -1223,8 +1223,8 @@ std::vector<at::Tensor> FusionExecutor::runFusion(
   if (isDebugDumpEnabled(DebugDumpOption::KernelArgs)) {
     std::cout << "Arguments for kernel" << fusion_id_ << ":" << std::endl
               << "Inputs:" << std::endl;
-    for (auto i : c10::irange(args.size())) {
-      args[i]->print();
+    for (auto i : c10::irange(num_inputs)) {
```
Comment (jjsjann123) [inline torch/csrc/jit/codegen/cuda/executor.cpp:1226]: Haha, thx for this. you beat me to it. https://github.com/pytorch/pytorch/issues/86529#issuecomment-1299115715

File: torch/csrc/jit/codegen/cuda/test/test_gpu_view.cpp [hunk 2]
```diff
@@ -2168,7 +2168,7 @@ TEST_F(NVFuserTest, FusionIssue2076_CUDA) {
   auto t12 = t4 + t11;
   auto t13 = t12.view({48, 128, 128});
   // 48, 128, 128
-  auto t14 = std::get<0>(t13.max({2}));
+  auto t14 = std::get<0>(t13.max(2));
```
Comment (jjsjann123) [inline torch/csrc/jit/codegen/cuda/test/test_gpu_view.cpp:2171]: wasn't this a compile time error before the fix?!

**Review by jjsjann123 — APPROVED** [review]
LGTM~ thx for fixing~~~

File: torch/csrc/jit/codegen/cuda/test/test_gpu_view.cpp (hunk 2 above)
Comment (naoyam) [reply torch/csrc/jit/codegen/cuda/test/test_gpu_view.cpp:2171]: It's not an error, but the parameter of `max` is just a scalar value, so clang was giving a warning that `{}` shouldn't be used.

Comment (naoyam) [reply torch/csrc/jit/codegen/cuda/test/test_gpu_view.cpp:2171]:
```
../torch/csrc/jit/codegen/cuda/test/test_gpu_view.cpp:2171:34: warning: braces around scalar initializer [-Wbraced-scalar-init]
  auto t14 = std::get<0>(t13.max({2}));
```

**Pull request merged**